articles/api-auth/tutorials/silent-authentication.md

```diff
@@ -1,45 +1,87 @@
+There are two main participants involved in a [single sign-on (SSO)](/sso) scenario: an Authorization Server (Auth0), and multiple client applications.
 
-## Access token expiration
+For privacy reasons, client applications cannot query Auth0 directly to determine if a user has logged in via SSO. This means that users must be redirected to Auth0 for SSO authentication.
 
-Access tokens are opaque to clients. This means that clients are unable to inspect the contents of access tokens to determine their expiration date.
+However, redirecting users away from your application is usually considered disruptive and should be avoided, from a UX perspective. **Silent authentication** lets you perform an authentication flow where Auth0 will only reply with redirects, and never with a login page.
 
-There are two options to determine when an access token expires:
+## Initiate a Silent Authentication request
+
+To initiate a silent authentication request, add the `prompt=none` parameter when you redirect a user to the [`/authorize` endpoint of Auth0's authentication API](/api/authentication#authorize-client).
+
+For example:
+
+```
+GET https://${account.namespace}/authorize
+    ?response_type=code&
+    client_id=...&
+    redirect_uri=...&
+    state=...&
+    scope=openid...&
+    prompt=none
+```
+
+__NOTE__: The specific parameters on the authentication request will depend on what kind of application is authenticating ([regular web application](/api/authentication#authorization-code-grant), [single page app](/api/authentication#implicit-grant), and so forth).
+
+The `prompt=none` parameter will cause Auth0 to immediately redirect to the specified `redirect_uri` (callback URL) with two possible responses:
+
+* A successful authentication response if the user was already logged in via SSO
+* An error response if the user is not logged in via SSO and therefore cannot be silently authenticated
 
-1. Read the [`expires_in` response parameter](https://openid.net/specs/openid-connect-core-1_0.html#ImplicitAuthResponse) returned by Auth0.
-2. Ignore expiration dates altogether. Instead, try to renew the access token if rejected by the API.
+### Successful authentication response
 
-The `expires_in` parameter is returned by Auth0 as a hash parameter following a successful authentication.
-For example, Auth0 could call your callback URL with the following parameters:
+If the user was already logged in via SSO, Auth0 will respond exactly as if the user had authenticated manually through the SSO login page.
 
-```text
-https://myapp.example.com/
-    #access_token=...&
-    id_token=...&
-    token_type=Bearer&
+For example, when using the [Authorization Code Grant](/api-auth/grant/authorization-code) (`response_type=code`, used for regular web applications), Auth0 will respond with an authorization code that can be exchanged for an ID token and optionally an access token:
```

Comment:
@rolodato Isn't this the other way around? -> `Auth0 will respond with an authorization code that can be exchanged for an ID token and optionally an access token`